simple/test.js

```diff
@@ -127,18 +127,17 @@ exports.test = function (global) {
 
   const time = internal.time;
   const print = internal.print;
-  
+
   // Substring first 5 characters to limit to A.B.C format and not use any `nightly`, `rc`, `preview` etc.
-  const serverVersion = (((typeof arango) !== "undefined") ? arango.getVersion() : internal.version).split("-")[0]
+  const serverVersion = (((typeof arango) !== "undefined") ? arango.getVersion() : internal.version).split("-")[0];
   const isEnterprise = internal.isEnterprise();
-  const isCluster = semver.satisfies(serverVersion, "<3.5.0") ? require("@arangodb/cluster").isCluster() : internal.isCluster();
+  const isCluster = internal.isCluster();
 
   print(`Running against version ${serverVersion} ${isEnterprise ? "Enterprise" : ""} ${isCluster ? "Cluster" : ""}`);
-  print(db._version(true));  
-  const supportsAnalyzers = !semver.satisfies(serverVersion,
-    "3.5.0-rc.1 || 3.5.0-rc.2 || 3.5.0-rc.3");
-  const supportsSatelliteGraphs = semver.satisfies(serverVersion, ">=3.7.0-devel");
-  const supportsOnlySplicedSubqueries = semver.satisfies(serverVersion, ">=3.9");
+  print(db._version(true));
+  const supportsAnalyzers = true;
+  const supportsSatelliteGraphs = true;
+  const supportsOnlySplicedSubqueries = false;
```

Comment:
We only support spliced subqueries since v3.9.0!
```suggestion
  const supportsOnlySplicedSubqueries = true;
```